manifests/setup/fluentbit-operator-deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
           mountPath: /var/run/docker.sock
       containers:
       - name: fluentbit-operator
-        image: 'kubespheredev/fluentbit-operator:v0.3.0'
+        image: 'kubesphere/fluentbit-operator:v0.3.0'
```

Comment:
I think maybe we can upgrade the version to v0.4.0

Reply:
Ok, i will release a new version after this pr is megerd and then change this.